config/database.yml

```diff
@@ -16,6 +16,9 @@ development:
 
 test:
   <<: *default
+  host: localhost
+  username: email-alert-api
+  password: email-alert-api
```

Comment:
Is this for CI? I wonder if it's better if we set this via DATABASE_URL env var? eg how we do this: https://github.com/alphagov/whitehall/blob/master/config/database.yml#L18 and set it in in the jenkinsfile (otherwise this may make it a bit of a pain to run tests locally)